src/include/sof/lib/perf_cnt.h

```diff
@@ -48,15 +48,15 @@ struct perf_cnt_data {
 	#ifdef CONFIG_TIMING_FUNCTIONS
 		#define perf_cnt_get_cpu_ts arch_timing_counter_get
 	#else
-		#define perf_cnt_get_cpu_ts k_cycle_get_64
+		#define perf_cnt_get_cpu_ts sof_cycle_get_64
 	#endif	/* CONFIG_TIMING_FUNCTIONS */
 #else
 	#define perf_cnt_get_cpu_ts() timer_get_system(cpu_timer_get())
 #endif	/* __ZEPHYR__ */
 
 /** \brief Initializes timestamps with current timer values. */
 #define perf_cnt_init(pcd) do {						\
-		(pcd)->plat_ts = k_cycle_get_64();			\
+		(pcd)->plat_ts = sof_cycle_get_64();			\
```

Comment:
Here "plat_ts" is the less accurate clock and this interface is used to measure events of much less than a millisecond, so 32bit accuracy would do. It's just that the interface is a 64bit value, so k_cycle_get_64() was just more practical.